JobRouterOpenAIIntegration/JR_AOAI_Integration/Program.cs

```diff
@@ -0,0 +1,222 @@
+            }
+        }
+
+        private static async Task ProvisionQueuesAndPolicies()
+        {
+            var distributionPolicy = await CreateDistributionPolicyAsync(
+                Configuration["JobRouterResources:DistributionPolicy:id"], 
+                Configuration["JobRouterResources:DistributionPolicy:name"], 
+                new TimeSpan(0, 5, 0));
+            Console.WriteLine($"{distributionPolicy.Name} was created");
+
+            var queue = await CreateQueueAsync(
+                Configuration["JobRouterResources:Queue:id"],
+                Configuration["JobRouterResources:Queue:name"],
+                distributionPolicy.Id);
+            Console.WriteLine($"{queue.Name} was created");
+        }
+
+        private static async Task<DistributionPolicy> CreateDistributionPolicyAsync(string id, string name, TimeSpan offerTTL)
+        {
+            try
+            {
+                var distributionPolicyResponse = await JobRouterAdministrationClient.CreateDistributionPolicyAsync(new CreateDistributionPolicyOptions(id, offerTTL, SetBestWorkerOpenAiMode())
+                {
+                    Name = name
+                });
+
+                return distributionPolicyResponse.Value;
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine($"Error occurred while creating distribution policy: {ex}");
+                throw;
+            }
+        }
+
+        private static async Task<RouterQueue> CreateQueueAsync(string id, string name, string distributionPolicyId)
+        {
+            try
+            {
+                var queueResponse = await JobRouterAdministrationClient.CreateQueueAsync(new CreateQueueOptions(id, distributionPolicyId) 
+                { 
+                    Name = name 
+                });
+
+                return queueResponse.Value;
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine($"Error occurred while creating queue: {ex}");
+                throw;
+            }
+        }
+
+        private async static Task<string> CreateJob()
+        {
+            Console.WriteLine("Creating a job...");
+
+            var jobOptions = new CreateJobOptions(Guid.NewGuid().ToString(), "voice", Configuration["JobRouterResources:Queue:id"]);
+            await JobRouterClient.CreateJobAsync(jobOptions);
+
+            Console.WriteLine("Job created successfully.");
+
+            return jobOptions.JobId;
+        }
+
+        private static BestWorkerMode SetBestWorkerOpenAiMode()
+        {
+            var mode = new BestWorkerMode() { ScoringRule = new FunctionRouterRule(new Uri(Configuration["OpenAiPairing:AzureFunctionUri"]))
+            {
+                Credential = new FunctionRouterRuleCredential(Configuration["OpenAiPairing:AzureFunctionKey"])
+            },
+                MinConcurrentOffers = 1,
```

Comment:
Min/Max default to 1, so they do not need to be set

Reply:
Removed